htmresearch/regions/TMRegion.py

```diff
@@ -117,22 +116,32 @@ def getSpec(cls):
         ),
         outputs=dict(
             bottomUpOut=dict(
-              description="""The primary output of the temporal memory.""",
+              description="""The default output of TMRegion. By default this
+                   outputs the active cells. You can change this dynamically
+                   using the defaultOutputType parameter.
+                   """,
```

Comment:
@subutai: an additional nitpick on top of Alex's - the text wrapped thusly doesn't print nicely (it even rhymes! :) )

```
In [3]: s = """abc
             def
             geh
             """

In [4]: print s
abc
             def
             geh
```